src/coverlet.core/Coverage.cs

```diff
@@ -232,14 +232,14 @@ private void CalculateCoverage()
                 }
 
                 // for MoveNext() compiler autogenerated method we need to patch false positive (IAsyncStateMachine for instance) 
-                // we'll remove all MoveNext() not covered branch              
+                // we'll remove all MoveNext() not covered branch
                 foreach (var document in result.Documents)
                 {
                     List<KeyValuePair<(int, int), Branch>> branchesToRemove = new List<KeyValuePair<(int, int), Branch>>();
                     foreach (var branch in document.Value.Branches)
                     {
                         //if one branch is covered we search the other one only if it's not covered
-                        if (CecilSymbolHelper.IsMoveNext(branch.Value.Method) && branch.Value.Hits > 0)
```

Comment:
I mean this method, you don't use it anymore and it was only used here, so can be removed. the regex can maybe remain there (it has the same name, so maybe my comment wasn't completely understood :P)

Reply:
I did it...I edited my last message!Thank's again!

Reply:
check, thanks! :D